src/Toolkit/Str.php

```diff
@@ -177,8 +194,11 @@ public static function after(string $string, string $needle, bool $caseInsensiti
      */
     public static function ascii(string $string): string
     {
-        $foreign = static::$ascii;
-        $string  = preg_replace(array_keys($foreign), array_values($foreign), $string);
+        foreach ([static::$language, static::$ascii] as $rules) {
+            $string  = preg_replace(array_map(function ($key) {
```

Comment:
I'm wondering if we could improve performance here by defining each character separately instead of using the pipe as separator and then replacing by regex. `preg_replace()` is a bit slower than `str_replace()` and because of the many rules (thousands in Chinese for example), this could become a bottleneck.

Reply:
For the slug rules, each is defined individually. Maybe a compromise would be to run the ASCII ones as is and switch to `str_replace` for th slug language rules?

Reply:
Yeah, that could be a good compromise. The ASCII ones aren't that many anyway. :)